db/seeds.rb

```diff
@@ -65,21 +65,26 @@
 
 # CUSTOMER
 150.times do
-    add_dog = Dog.order("RANDOM()").take
-
-    if !Customer.exists?(dog_id: add_dog)
-        new_customer = Customer.create(
-            first_name: Faker::Name.first_name,
-            last_name: Faker::Name.last_name,
-            age: Faker::Number.between(from: 1, to: 100),
-            dog_id: add_dog.id
-        )
+    set_rand_dog_count = rand(0..10)
```

Comment:
what does this do??

Reply:
It generates a random number from 0-10 to be used for giving how many dogs a customer has.